tests/acceptance/blockchain/actors/test_ursula_operator.py

```diff
@@ -26,169 +17,6 @@ def log(message):
     print(message)
 
 
-@pytest.mark.skip()
-@pytest_twisted.inlineCallbacks
-def test_work_tracker(
```

Comment:
Is there another test for operator bonded tracking?

Reply:
If I'm understanding the question correctly.

This specific test function focused on ongoing commitments (multiple periods) and replacement transactions for legacy PRE, which is less of an issue on Threshold. There are 2 existing tests remaining in this file: one for testing the confirmation function, the other for auto-confirmation on Ursula start-up. Additionally, there are WorkTracker unit tests - tests/unit/test_work_tracker_error_handling. So this is all related to having the node confirm its operator.

Operator bonded tracking ensures that the node will shut down if it is ever unbonded. Tests for that are done via tests/unit/test_operator_bonded_tracker.py

Reply:
Are you thinking we instead adapt this test (instead of removing it) to continue testing replacement transactions? Multiple periods is irrelevant now, so no need to test.

Reply:
I believe the stack is essentially paused inside the Ursula/Operator's init until the transaction is confirmed.  During this period (no pun intended) there may still be replacement transactions issued. So, generally I'm wondering if that's tested elsewhere.  I vaguely remember this test being skipped and raising my eyebrows.

Reply:
I think I've figured out a way to update/keep this test via https://github.com/nucypher/nucypher/pull/3034/commits/1b68b32fbe330d51e56ba68e105286655429e023